docs/Node Operators/node-versions.md

```diff
@@ -12,13 +12,25 @@ metadata:
 
 You can find a list of release notes for Chainlink nodes in the [smartcontractkit GitHub repository](https://github.com/smartcontractkit/chainlink/releases). Docker images are available in the [Chainlink Docker hub](https://hub.docker.com/r/smartcontract/chainlink/tags).
 
-## Upgrading to v1.0.0 nodes
+## Changes to v1.1.0 nodes
+
+**[v1.1.0 release notes](https://github.com/smartcontractkit/chainlink/releases/tag/v1.1.0)**
+
+The v1.1.0 release includes several substantial changes to the way you configure and operate Chainlink nodes:
+
+- **Legacy environment variables**: Legacy environment variables are supported, but they might be removed in future node versions. See the [Configuring Chainlink Nodes](/docs/configuration-variables/#evmethereum-legacy-environment-variables) page to learn how to migrate your nodes away from legacy environment variables and use the API, CLI, or GUI exclusively to administer chains and nodes.
+- **Failover nodes**: The process for configuring failover nodes has changed and includes a new `DATABASE_LOCKING_MODE=lease` configuration variable. See the new [Failover Node Example](/docs/performing-system-maintenance/#failover-node-example) to learn the new best practices for configuring and testing failover nodes that share the same database.
```

Comment:
Should messaging be added to highlight `DATABASE_LOCKING_MODE=advisorylock` should be the default for 1.1.0?

Reply:
Updated the default and clarified best practices. See commit [32c596f](https://github.com/smartcontractkit/documentation/pull/558/commits/32c596f5365545fc2d3855a82d396c4cef6faf81) for the delta.

Reply:
I removed the Failover nodes entry for now.